src/components/Reactions/AddReactionBubble.js

```diff
@@ -99,4 +110,11 @@ AddReactionBubble.propTypes = propTypes;
 AddReactionBubble.defaultProps = defaultProps;
 AddReactionBubble.displayName = 'AddReactionBubble';
 
-export default withLocalize(AddReactionBubble);
+export default compose(
+    withLocalize,
+    withOnyx({
+        session: {
+            key: ONYXKEYS.SESSION,
+        },
+    }),
```

Comment:
I think we can move this to libs/actions/Session/index.js as well so that we don't need to add this code in every component which calls `Session.checkIfActionIsAllowed`.

Reply:
Oh also good catch, and yeah makes sense to have this in the lib than loading in every component. Thanks! Updating

Reply:
Also it makes the PR easier to review. Updated!